docs/src/pages/styles/basics/basics.md

```diff
@@ -202,3 +202,21 @@ const useStyles = makeStyles(theme => ({
 ```
 
 {{"demo": "pages/styles/basics/StressTest.js"}}
+
+## @material-ui/styles vs @material-ui/core/styles
+
+Material-UI styles are powered by the [@material-ui/styles](https://www.npmjs.com/package/@material-ui/styles) npm package (and JSS).
+This solution is [isolated](https://bundlephobia.com/result?p=@material-ui/styles), it does not know the default Material-UI theme.
+It can be used to style React applications that are not using Material-UI framework.
+
+To remove the need for injecting a theme in the React's context **systematically** and to install multiple dependencies, the `@material-ui/styles` modules are re-exported from `@material-ui/core/styles` (with a default theme).
```

Comment:
This sentence isn't clear. The middle needs to be re-written. Reading this, it almost seems that we did it, so multiple dependencies are installed - not to reduce the number of manual installations a developer needs to do.

Reply:
It comes with #17419.